src/showcase/src/pages/flows/[flow].astro

```diff
@@ -1,10 +1,10 @@
 ---
 import Layout from "../../layouts/Layout.astro";
 import "$src/styles/main.css";
-import { iiFlows } from "$showcase/flows";
 
+export const iiFlowNames = ["loginManage", "register"];
 export function getStaticPaths() {
-  return Object.keys(iiFlows).map((key) => ({ params: { flow: key } }));
+  return iiFlowNames.map((name) => ({ params: { flow: name } }));
```

Comment:
Maybe you can use another variable then `name`. It's commonly advised to not use such a name.

https://www.w3schools.com/js/js_reserved.asp

> JavaScript Objects, Properties, and Methods
> You should also avoid using the name of JavaScript built-in objects, properties, and methods: 
> ...
> name

Maybe `key`